std/format.d

```diff
@@ -6055,3 +6018,10 @@ char[] sformat(Char, Args...)(char[] buf, in Char[] fmt, Args args)
     tmp = format("%19,?f", '.', -1234567.891011);
     assert(tmp == " -1.234.567.891.011", "'" ~ tmp ~ "'");
 }
+
+// Test for multiple indexes
```

Comment:
Oops, went on vacations and just cleared my notifications when I returned. Did you submit a P.R. ?